src/Appium.Net/Appium.Net.csproj

```diff
@@ -26,15 +26,28 @@
     <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <LangVersion>8.0</LangVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard2.0|AnyCPU'">
+    <NoWarn>1701;1702;1591</NoWarn>
+  </PropertyGroup>
+
   <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|netstandard2.0|AnyCPU'">
-    <DocumentationFile>Appium.Net.xml</DocumentationFile>
+    <NoWarn>1701;1702;1591</NoWarn>
```

Comment:
@akinsolb can you explain what is the reason we need it?

Reply:
The error code reduces the warnings on build. The xml file is used by consumers to see the documentation of the library. Currently this isn't included in the drop folder as we set the the file name. Removing it places it back in the default path, the build output directory.